network/load-balancer/quickstart.mdx

```diff
@@ -6,78 +6,86 @@ content:
-        - Server IPs: enter the IP addresses of each of your backend servers.
-    - Configure **advanced settings** if required:
-      - **S3 failover**: activate the feature and enter a Scaleway Object Storage [Bucket Website](/storage/object/concepts/#bucket-website) URL. If none of your backend servers are available, users will be redirected to this page. 
+      - `WAW`: Warsaw, Poland.
+
+5. Select a Load Balancer **type**, according to your bandwidth needs.
+    <Lightbox src="scaleway-select-load-balancer.webp" alt="" />
+
+6. Select an **IP address** for your Load Balancer. You can either allocate a new IP, or select one of your existing IPs.
+
+7. Configure your Load Balancer's **[frontend](/network/load-balancer/concepts/#frontends)** by entering a **name** and a **port**. The frontend will listen on this port, and forward the requests it receives from clients to the backend(s). Note that you cannot add an SSL certificate at this stage, but you can [add one after the creation of the Load Balancer](/network/load-balancer/how-to/add-certificate/). You can also add [additional frontends](#how-to-add-frontends) after creation.
+    <Lightbox src="scaleway-configure-frontend.webp" alt="" />
 
+8. Configure your Load Balancer's **[backend](/network/load-balancer/concepts/#backends)** by entering a **name**, and the **protocol** and **port** it should use to connect to the backend servers. Enter also a **proxy**, **health check** method and the **server IPs** of the backend servers it should direct traffic to. You can [add additional backends](#how-to-add-backends) after creating your Load Balancer, if required.
+    <Lightbox src="scaleway-configure-backend.webp" alt="" /> 
+    Depending on the options you choose, you may be prompted for additional configurations such as the **Verify certificate** setting if you select the HTTPS protocol. For more information on choosing configuration settings for your Load Balancer's backend, see our [dedicated reference documentation](/network/load-balancer/reference-content/configuring-load-balancer)
+
+9. Configure **advanced settings** if required:
+    - **S3 failover**: activate the feature and enter a Scaleway Object Storage [Bucket Website](/storage/object/concepts/#bucket-website) URL. If none of your backend servers are available, users will be redirected to this page. 
       <Message type="note">
         The S3 failover feature is only available for Load Balancers using the HTTP protocol on their backend. Find out more about setting up a static website with Object Storage in our [dedicated documentation](/storage/object/how-to/use-bucket-website/).
-
       </Message>
+    - **[Sticky session](/network/load-balancer/concepts/#sticky-session)**: activate the feature and (if relevant) enter the name for a cookie that will bind users' sessions to specific backend servers.
```

Comment:
Maybe a bit more info could help here like: "Use this feature if you want a particular user session to stick to the same backend server. This is useful if your users need their requests to be served by a single server in order to maintain a consistent session".

Reply:
done